docs/known_issues.md

```diff
@@ -14,10 +14,23 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## `form_for` compatibility
```

Comment:
There are actually two form helpers affected, `form_for` and `form_with`. I think it makes sense to refer to them both together as "form helpers."

```suggestion
## Compatibility with Rails form helpers
```